### DISCUSSION TITLE
Installing the module for the powershell script

### DISCUSSION THREAD

**Discussion by xexe73** [community/community#32052] [Other Feature Feedback, Questions, & Ideas]
Hello everyone. Please tell me how in github action, when installing a module for powershell, the default value is passed "N", how to change it to "Y"?
![изображение](https://user-images.githubusercontent.com/104350800/188501021-e24ea8c0-87f2-43ac-8bcd-290d035503ff.png)

**Comment by Ba4bes** [comment]
On a hosted Agent you can use two methodes:

The Force parameter:
```PowerShell
Install-Module ModuleName -Force  
```
Or set the PSGallery to trusted before you install the module:
```PowerShell
Set-PSRepository -Name 'PSGallery' -InstallationPolicy Trusted
Install-Module ModuleName
```

Both have pretty much the same result when you run it in a GitHub Actions workflow
